test/integration/Main.hs

```diff
@@ -1 +1,66 @@
-{-# OPTIONS_GHC -F -pgmF hspec-discover #-}
+module Main where
+
+import Control.Concurrent
+    ( threadDelay )
+import Control.Concurrent.MVar
+    ( newMVar )
+
+import Data.Aeson
+    ( Value )
+import Data.String
+    ( fromString )
+import Data.Text
+    ( Text )
+import Network.HTTP.Client
+    ( Manager, defaultManagerSettings, newManager )
+import Prelude
+import System.Process
+    ( proc, withCreateProcess )
+import Test.Hspec
+    ( beforeAll, describe, hspec )
+
+import qualified Data.Text as T
+
+import Test.Integration.Framework.DSL
+    ( Context (..)
+    , Scenarios
+    , expectError
+    , request
+    , request_
+    , scenario
+    , verify
+    )
+import Test.Integration.Framework.Request
+    ( RequestException (..) )
+
+import qualified Cardano.NetworkLayer.HttpBridgeSpec as HttpBridge
+
+withWallet :: ((Text, Manager) -> IO a) -> IO a
+withWallet action = do
+    let launch = proc "cardano-wallet-server" []
```

Comment:
Might want to use `Command` from `Launcher` instead of `proc` here, but shouldn't matter

Reply:
Ah, right, this is a different situation than the one in `Cardano.NetworkLayer.HttpBridgeSpec`. Nvm.

Reply:
This is running `cardano-wallet-server` and not `cardano-wallet-launcher` though 🤔

Reply:
At this stage, `withWallet` is more of a placeholder because there is no API server yet. I just have it spawning some server process then run the test cases.